src/backend/serial/u32/scalar.rs

```diff
@@ -13,12 +13,16 @@
 use core::fmt::Debug;
 use core::ops::{Index, IndexMut};
 
+use zeroize::DefaultIsZeroes;
+
 use constants;
 
 /// The `Scalar29` struct represents an element in ℤ/lℤ as 9 29-bit limbs
-#[derive(Copy,Clone)]
+#[derive(Copy, Clone, Default)]
 pub struct Scalar29(pub [u32; 9]);
 
+impl DefaultIsZeroes for Scalar29 {}
```

Comment:
This could perhaps be an explicit `Zeroize` impl instead of adding a `Default` bound.